examples/remote_control_cozmo.py

```diff
@@ -32,11 +32,41 @@
     sys.exit("Cannot import from flask: Do `pip3 install --user flask` to install")
 
 try:
-    from PIL import Image
+    from PIL import Image, ImageDraw
 except ImportError:
     sys.exit("Cannot import from PIL: Do `pip3 install --user Pillow` to install")
 
 
+# Annotator for displaying RobotState (position, etc.) on top of the camera feed
+class RobotStateDisplay(cozmo.annotate.Annotator):
+    def apply(self, image, scale):
+        d = ImageDraw.Draw(image)
+
+        bounds = [3, 0, image.width, image.height]
+
+        def print_line(text_line):
+            nonlocal bounds
+            text = cozmo.annotate.ImageText(text_line, position=cozmo.annotate.TOP_LEFT, color='lightblue')
+            text.render(d, bounds)
+            TEXT_HEIGHT = 11
+            bounds[1] += TEXT_HEIGHT
+
+        robot = self.world.robot
+
+        # Display the Pose info for the robot
+
+        pose = robot.pose
+        print_line('Pose: Pos = <%.1f, %.1f, %.1f>' % pose.position.x_y_z)
+        print_line('Pose: Rot quat = <%.1f, %.1f, %.1f, %.1f>' % pose.rotation.q0_q1_q2_q3)
+        print_line('Pose: angle_z = %.1f' % pose.rotation.angle_z.degrees)
+        print_line('Pose: origin_id: %s' % pose.origin_id)
+
+        # Display the Accelerometer and Gyro data for the robot
+
+        print_line('Acclrm: <%.1f, %.1f, %.1f>' % robot.accelerometer.x_y_z)
```

Comment:
"Accelerometer" takes up a lot of space in the image, and I was worried that "Acc" might be confusing (as in "why does it show Cozmo accelerating when he's stationary", and apparently "Acclrm" is the abbreviation for that, but i agree it looks awkward. I'll go with "Accel"?